### PR TITLE
Update doc with kube_init_skip param

### PR DESCRIPTION
**Pull request by minhdanh**
Update doc as a new param is introduced to plugin pelotech/drone-helm3 in this PR: https://github.com/pelotech/drone-helm3/pull/102

### CODE REVIEW

**Comment by minhdanh** [comment]
https://github.com/pelotech/drone-helm3/pull/102 has been merged

File: content/pelotech/drone-helm3/index.md [hunk 1]
```diff
@@ -65,20 +65,44 @@ string_values
 values_files
 : Arguments for helm's `--values` flag.
 
+skip_kubeconfig
+: Whether to skip kubeconfig file creation.
+: If you already have a kubeconfig file (like in a temporary volume shared between steps), you can skip the creation of kubeconfig file.
+: For example:
+
+```yaml
```
Comment (tboerger) [inline content/pelotech/drone-helm3/index.md:73]: We are not adding code blocks to the parameter list.

Comment (minhdanh) [reply content/pelotech/drone-helm3/index.md]: @tboerger Thanks. Then I removed the block. Please check. Thanks!

**Comment by minhdanh** [comment]
@tboerger Can check this PR again please?

**Review by tboerger — APPROVED** [review]

**Pull request closed without merging**